test/classes/Gis/GisGeomTestCase.php

```diff
@@ -62,4 +66,28 @@ public function testScaleRow(string $spatial, array $min_max): void
             $this->object->scaleRow($spatial)
         );
     }
+
+    /**
+     * Create a new pdf document with predictable timestamps and ids.
+     *
+     * @param string $id Used as a seed for the internal file_id
+     *
+     * @return TCPDF A pdf document with an empty page
+     */
+    protected function createEmptyPdf(string $id): TCPDF
+    {
+        $pdf = new TCPDF();
```

Comment:
since tcpdf is optional, we should add a 
```suggestion

        if (! class_exists(TCPDF::class)) {
            $this->markTestSkipped('TCPDF is missing');
        }
        $pdf = new TCPDF();
```

Reply:
TCPDF is listed under `require-dev`. Wouldn't that always make it available for tests?
https://github.com/phpmyadmin/phpmyadmin/blob/22ab240b3316b0dae04db346e654b975266af8bb/composer.json#L97-L113